Readme.md

```diff
@@ -3,3 +3,48 @@
+
+The [LayoutVersion](https://docs.devexpress.com/CoreLibraries/DevExpress.Utils.OptionsLayoutBase.LayoutVersion) property is defined both for two Pivot Grid controls. When you click the "Restore layout" button and the saved layout is restored from a data store, the **LayoutUpgrade** event is raised because the layout versions are different. As a result, a new "Quantity" field is added to the new Pivot Grid control.
+
+The Pivot Grid uses a field's [Name](https://docs.devexpress.com/CoreLibraries/DevExpress.XtraPivotGrid.PivotGridFieldBase.Name) property value to determine fields in a stored layout.
+
+API in this example:
+* [ASPxPivotGrid.SaveLayoutToStream](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPivotGrid.ASPxPivotGrid.SaveLayoutToStream(System.IO.Stream))
+
+    Saves the Pivot Grid layout to the specified stream.
+    
+* [ASPxPivotGrid.LoadLayoutFromStream](https://docs.devexpress.com/AspNet/DevExpress.Web.ASPxPivotGrid.ASPxPivotGrid.LoadLayoutFromStream(System.IO.Stream))
+
+    Restores the Pivot Grid layout from the specified stream.
+
+* [OptionsColumnLayout.StoreAppearance](https://docs.devexpress.com/CoreLibraries/DevExpress.Utils.OptionsColumnLayout.StoreAppearance)
+
+    Gets or sets whether the appearance settings of the columns and bands are also stored when the layout is saved to storage and restored when the layout is restored from storage. In this example, stores the caption of the "Year" and "Quarter" fields.
+
+* [OptionsColumnLayout.AddNewColumns](https://docs.devexpress.com/CoreLibraries/DevExpress.Utils.OptionsColumnLayout.AddNewColumns), [OptionsColumnLayout.RemoveOldColumns](https://docs.devexpress.com/CoreLibraries/DevExpress.Utils.OptionsColumnLayout.RemoveOldColumns)
+    > New columns are fields in the control, old columns are fields in the saved layout.
+
+    If old and new fields have the same ID, the control applies the saved layout changes to the fields in the control.
+    
+    If old and new fields have the different ID, the behavior is following: 
+
+    - If both properties are *true*, Pivot Grid removes fields from the saved layout and leaves fields in the control. 
```

Comment:
Are you saying:
If both properties are *true*, the Pivot Grid removes fields from the saved layout and restores these fields in the control. 
Or are you saying:
If both properties are *true*, the Pivot Grid removes fields from the saved layout and allows the fields in the control to remain there. 
-not sure if the word "these" that I added is appropriate here; please review